From 7cac2341ba7e2419267a7e9ee5e5871de744bedb Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ronald=20Tschal=C3=A4r?= <ronald@innovation.ch>
Date: Tue, 16 May 2017 18:49:14 -0700
Subject: [PATCH] Recognize nvme disks as such too.

The pattern is based on name assignment in nvme_alloc_ns() in the linux
kernel in drivers/nvme/host/core.c .
---
 photosyst.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/photosyst.c b/photosyst.c
index 38828ec..3d38df5 100644
--- a/photosyst.c
+++ b/photosyst.c
@@ -1537,6 +1537,7 @@ static struct {
 	{ "^dasd[a-z][a-z]*$",			{0},  nullmodname, DSKTYPE, },
 	{ "^mmcblk[0-9][0-9]*$",		{0},  nullmodname, DSKTYPE, },
 	{ "^emcpower[a-z][a-z]*$",		{0},  nullmodname, DSKTYPE, },
+	{ "^nvme[0-9][0-9]*n[0-9][0-9]*$",	{0},  nullmodname, DSKTYPE, },
 };
 
 static int
